DataFormats/Reconstruction/include/ReconstructionDataFormats/TrackFwd.h

```diff
@@ -55,9 +57,24 @@ class TrackParFwd
   void setPhi(Double_t phi) { mParameters(2) = phi; }
   Double_t getPhi() const { return mParameters(2); }
 
+  Double_t getSnp() const
```

Comment:
Hi Rita,

Thanks for the PR. I have some small comments. For the bulk of the PR (`FwdDCAFitterN.h`)  I'll let more knowledgable people comment on...  

Concerning `getSnp` and `getCsp2` I'd have a couple of remarks :

- given that they use data (phi) that are publicly available (getPhi) I don't see the point of making those member functions (I would see the point if they were internally cached or something, but that's not the case here)  

- why use such cryptic names ? Might be obvious to you, but honestly when I see `getSnp` I don't immediately understand `getSinPhi` ;-)  Would matter less in an implementation file, but here we are at the interface level.

Reply:
Hi Laurent, these methods and names were added to keep symmetry with [central barrel track classes](https://github.com/AliceO2Group/AliceO2/blob/dev/DataFormats/Reconstruction/include/ReconstructionDataFormats/TrackParametrization.h#L150-L160) and allow their use in template classes, such as HelixHelper. This is not the (yet) the case for `getSnp` and `getCsp2`. Rita could rename/remove these two in this PR, but this would only increase entropy by diverging from central barrel, unless this is also renamed in other classes in ReconstructionDataFormat.